src/core/validators/validators.ts

```diff
@@ -31,7 +33,7 @@ const EmptyString: ValidatorFn = (value) => {
   console.warn(
     'Calling a deprecated validator (EmptyString), please use the Required validator instead'
   );
-  if (isEmpty(value)) {
+  if (isEmptyValue(value)) {
```

Comment:
https://lodash.com/docs/4.17.15#isEmpty

Reply:
isEmptyValue() is also checking that the string doesn't contain spaces. 
The isEmpty() from lodash is not checking it.

Reply:
you may wanna trim it if possible before passing it ...